modules/post/networking/gather/enum_cisco.rb

```diff
@@ -0,0 +1,244 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/auxiliary/cisco'
+
+class MetasploitModule < Msf::Post
+  include Msf::Auxiliary::Cisco
+  include Msf::Exploit::Deprecated
+  moved_from 'post/cisco/gather/enum_cisco'
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Cisco Gather Device General Information',
+        'Description' => %q{
+          This module collects a Cisco IOS or NXOS device information and configuration.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [ 'Carlos Perez <carlos_perez[at]darkoperator.com>'],
+        'Platform' => [ 'cisco'],
+        'SessionTypes' => [ 'shell' ]
```

Comment:
Should there not have been a release date added for this module? Do we know how old this was?

Reply:
as old as time itself. https://github.com/rapid7/metasploit-framework/commit/0859c99940ab6b9c4d63ffe6ffadfbd28c9b183c#diff-58ffe124367013d6563077cb97591f74
Do you mean `DisclosureDate`? This is just using admin functionality, no exploitation, so no disclosure date.

Reply:
Ah ok yeah think I meant `DisclosureDate` but makes sense based off of what you said that this wouldn't have one. I'll close this as resolved, sorry for the confusion :)